pkgs/development/python-modules/bambi/default.nix

```diff
@@ -36,10 +34,8 @@ buildPythonPackage rec {
   propagatedBuildInputs = [
```

Comment:
Missing `graphviz` (which should be moved from `nativeCheckInputs` to here)

Reply:
Done